app/src/main/java/com/hyperion/ths/marvel_03/ui/main/MainViewPageAdapter.java

```diff
@@ -14,20 +15,23 @@
 
 /**
  * Created by ths on 27/05/2017.
+ * Thuan
  */
 
 public class MainViewPageAdapter extends FragmentStatePagerAdapter {
-    private Context mContext;
-    private String[] mTabsTitle = { "Hero", "Favorite" };
-    private int[] mTabsIcons = { R.drawable.ic_hero_selector, R.drawable.ic_favorite_selector };
+    private final Context mContext;
+    private final String[] mTabsTitle = { "Hero", "Favorite" };
+    private final int[] mTabsIcons =
+            { R.drawable.ic_hero_selector, R.drawable.ic_favorite_selector };
```

Comment:
ERROR: 'array initialization lcurly' have incorrect indentation level 12, expected level should be one of the following: 4, 8.